Table/src/lib/FitsWriterHelper.cpp

```diff
@@ -201,6 +201,19 @@ std::vector<std::valarray<T>> createNdArrayColumnData(const Euclid::Table::Table
   return result;
 }
 
+template <typename T>
+std::vector<T> createSingleNdArrayVectorColumnData(const Euclid::Table::Table& table, size_t column_index) {
+  std::vector<T> result {};
+  for (auto& row : table) {
+    const auto& nd = boost::get<NdArray<T>>(row[column_index]);
+    if (nd.size())
```

Comment:
would rather explicitly do 
if (nd.size() > 0) {

Reply:
Modified with the last commit